src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BookingService.kt

```diff
@@ -122,14 +124,46 @@ class BookingService(
   @Value("\${url-templates.frontend.application}") private val applicationUrlTemplate: String,
   @Value("\${url-templates.frontend.booking}") private val bookingUrlTemplate: String,
   @Value("\${arrived-departed-domain-events-disabled}") private val arrivedAndDepartedDomainEventsDisabled: Boolean,
+  private val userService: UserService,
+  private val userAccessService: UserAccessService,
+  private val bookingTransformer: BookingTransformer,
 ) {
   val approvedPremisesBookingAppealedCancellationReasonId: UUID =
     UUID.fromString("acba3547-ab22-442d-acec-2652e49895f2")
 
   private val log = LoggerFactory.getLogger(this::class.java)
 
   fun updateBooking(bookingEntity: BookingEntity): BookingEntity = bookingRepository.save(bookingEntity)
-  fun getBooking(id: UUID) = bookingRepository.findByIdOrNull(id)
+
```

Comment:
Just a question, `getBooking()` method is used by `getBookingForPremises` which is then used in multiple places in the premisesController. So not sure the additional function you added here are applicable to other places, i expect the test should fail to reflect if not then just a question.

Reply:
Hi - good spot. I addressed this in my first commit - https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1409/commits/330d6deaa0359db97351c4a8904e87d6b1cfdda8

The few services were using this are now using the repository directly to ensure i didn't break them

Reply:
oh got it, how did i missed it?!